build/tasks/minify.js

```diff
@@ -0,0 +1,65 @@
+"use strict";
+
+const swc = require( "@swc/core" );
+const fs = require( "fs" );
+const path = require( "path" );
+
+const rjs = /\.js$/;
+
+async function minify( { filename, folder, esm } ) {
+	const contents = await fs.promises.readFile( path.join( folder, filename ), "utf8" );
+	const version = /jQuery JavaScript Library ([^\n]+)/.exec( contents )[ 1 ];
+
+	const { code, map: incompleteMap } = await swc.minify(
+		contents,
+		{
+			compress: {
+				ecma: esm ? 2015 : 5,
+				hoist_funs: false,
+				loops: false
+			},
+			format: {
+				ecma: esm ? 2015 : 5,
+				asciiOnly: true,
+				comments: false,
+				preamble: `/*! jQuery ${version}` +
+					" | (c) OpenJS Foundation and other contributors" +
+					" | jquery.org/license */\n"
+			},
+			mangle: true,
+			inlineSourcesContent: false,
+			sourceMap: true
+		}
+	);
+
+	const minFilename = filename.replace( rjs, ".min.js" );
+	const mapFilename = filename.replace( rjs, ".min.map" );
+
+	const map = JSON.stringify( {
+		...JSON.parse( incompleteMap ),
+		file: minFilename,
+		sources: [ filename ]
+	} );
```

Comment:
Please re-add the comment from:
https://github.com/jquery/jquery/blob/f75daab09102a4dd5107deadb55d4a169f86254a/Gruntfile.cjs#L349-L351
here.